manifests/server/config.pp

```diff
@@ -188,6 +188,19 @@
 
   ## PuppetDB
   if $puppet::server_puppetdb_host {
+
+    unless $puppet::params::puppetdb_version == undef {
+      $pkg_suffix = $::osfamily ? {
+        'Debian'  => '-1puppetlabs1',
+        'FreeBSD' => '',
+        'Redhat'  => "-1.el${$puppet::params::osreleasemajor}",
+      }
+
+      class { '::puppetdb::globals':
+        version => "${puppet::params::puppetdb_version}${pkg_suffix}",
```

Comment:
I would argue against referencing ::params things outside of params and the init class. We should always let the user override.